quicksort/QuickSort.java

```diff
@@ -0,0 +1,36 @@
+public class QuickSort {
```

Comment:
@jsroyal What about adding a `package main` at the top. This will fix this issue.